src/file-input.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div class="vue-fine-uploader-file-input">
+    <label>
+      <slot>Upload File</slot>
```

Comment:
Might be a good idea to have 2 slots, one when `multiple === false` and one when `multiple === true`, thus we could have "Upload File" or "Upload Files", might be easier to customise the label.

This is open for discussion of course.